### ISSUE TITLE
Some Hydra sbt plugin error

### ISSUE THREAD

**Issue by travisbrown** [typelevel/cats#2870]
I have no idea what this is about but it's annoying. It's happening on current master on my machine.
```
$ sbt prePR
[info] Loading settings for project global-plugins-build from sbt-updates.sbt ...
[info] Loading global plugins from /home/travis/.sbt/1.0/plugins/project
[info] Loading settings for project global-plugins from sbt-updates.sbt ...
[info] Loading global plugins from /home/travis/.sbt/1.0/plugins
Waiting for lock on /home/travis/.ivy2/.sbt.ivy.lock to be available...
[info] Loading settings for project cats-build-build from plugins.sbt ...
[info] Loading project definition from /home/travis/code/projects/cats/project/project
[info] Loading settings for project cats-build from build.sbt,plugins.sbt ...
[info] Loading project definition from /home/travis/code/projects/cats/project
[info] Loading settings for project cats from build.sbt,version.sbt ...
[info] Resolving key references (23201 settings) ...
[info] Set current project to cats (in build file:/home/travis/code/projects/cats/)
[info] sbt started with 3818 MB memory heap
[info] Checking for Hydra latest version at https://repo.triplequote.com/artifactory/libs-release/...
[info] You are running the latest Hydra release.
[info] The Triplequote Hydra Scala compiler won't be used because the /home/travis/.triplequote/hydra.license is missing. Instead, compilation is carried out as usual with the vanilla Scala compiler.
[success] Total time: 17 s, completed Jun 1, 2019 7:56:45 AM
[success] Total time: 7 s, completed Jun 1, 2019 7:56:52 AM
[success] Total time: 1 s, completed Jun 1, 2019 7:56:53 AM
[info] Setting Scala version to 2.11.12 on 29 projects.
[info] Excluded 1 projects, run ++ 2.11.12 -v for more details.
[info] Reapplying settings...
[info] Set current project to cats (in build file:/home/travis/code/projects/cats/)
[info] sbt started with 3818 MB memory heap
[info] Checking for Hydra latest version at https://repo.triplequote.com/artifactory/libs-release/...
[info] You are running the latest Hydra release.
[info] The Triplequote Hydra Scala compiler won't be used because the /home/travis/.triplequote/hydra.license is missing. Instead, compilation is carried out as usual with the vanilla Scala compiler.
[error] java.util.NoSuchElementException
[error]         at scala.collection.LinearSeqOptimized.last(LinearSeqOptimized.scala:146)
[error]         at scala.collection.LinearSeqOptimized.last$(LinearSeqOptimized.scala:145)
[error]         at scala.collection.immutable.List.last(List.scala:85)
[error]         at com.triplequote.internal.sbt.hydra.compat.SbtCompatImpl$.lastRemainingCommand(SbtCompatImpl.scala:141)
[error]         at com.triplequote.sbt.hydra.command.HydraLocalDashboard$.maybeStartDashboard(HydraLocalDashboard.scala:107)
[error]         at com.triplequote.sbt.hydra.HydraPlugin$.$anonfun$globalSettings$6(HydraPlugin.scala:187)
[error]         at scala.Function1.$anonfun$andThen$1(Function1.scala:52)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:44)
...
```

**Comment by kailuowang** [comment]
This is a regression from the Hydra plugin,  it shouldn't throw an exception when the license is missing. It should've carried on to use the regular scalac. cc @dotta

**Comment by dragos** [comment]
Very sorry about this. Will cut a release with a fix ASAP (The regression is in 2.1.5, 2.1.4 fine).

**Comment by dragos** [comment]
2.1.6 is out with a fix.
